tests/components/plex/test_config_flow.py

```diff
@@ -89,21 +95,20 @@ def init_config_flow(hass):
         )
         assert result["type"] == "create_entry"
         assert result["title"] == mock_plex_server.friendlyName
-        assert result["data"][config_flow.CONF_SERVER] == mock_plex_server.friendlyName
-        assert (
-            result["data"][config_flow.CONF_SERVER_IDENTIFIER]
-            == mock_plex_server.machineIdentifier
-        )
-        assert result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_URL] == used_url
-        assert (
-            result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_TOKEN]
-            == file_config[CONF_TOKEN]
-        )
+        assert result["data"][
+            config_flow.CONF_SERVER] == mock_plex_server.friendlyName
+        assert (result["data"][config_flow.CONF_SERVER_IDENTIFIER] ==
+                mock_plex_server.machineIdentifier)
+        assert result["data"][
+            config_flow.PLEX_SERVER_CONFIG][CONF_URL] == used_url
+        assert (result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_TOKEN] ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4636425)